### ISSUE TITLE
[Bug Report] Custom Model Data within Cooldowns module not working on releases 1.20.5 and forward

### ISSUE THREAD

**Issue by jaymesss** [LunarClient/Apollo#173]
### Checklist before creating an issue:

- [X] I'm using the [latest version](https://github.com/LunarClient/Apollo/releases) of Apollo.
- [X] I'm using the latest version of Lunar Client. (Relaunched in the last hour)
- [X] I'm running Apollo on **ONLY** the backend server OR the proxy. **(YOU CANNOT RUN IT ON BOTH)**
- [X] I do **NOT** have both Apollo and the LEGACY API installed on the same server. **(YOU CANNOT RUN BOTH AT THE SAME TIME)**
- [X] I've gathered all screenshots, logs and appropriate information related to creating a bug report.

### Issue Description

ItemStackIcon with set custom model data does not render in-game on the cooldowns hud with the model data on any version greater than 1.20.4. First image attached is 1.21.1, second image is 1.20.4.
![image](https://github.com/user-attachments/assets/ecd79698-c3f9-46d6-82bf-d8253eb49729) 
![image](https://github.com/user-attachments/assets/6ba2fb37-10ff-44d4-baff-e4391c7f8a3f)

### Reproduction steps

```bash
1. Create an ItemStackIcon with custom model data set.
2. Present the cooldown to the player running 1.20.5 or greater
```

### Code Snippet (If applicable, if not write N/A)

```java
ItemStackIcon grapplingIcon = ItemStackIcon.builder()
        .itemName(grapplingHook.material().name())
        .customModelData(grapplingHook.modelData())
        .build();
```

### Versions

1.20+

### Platforms

Spigot / Bukkit

### Server Version

1.20+

### Modules

Fabric

### Contact Information

```bash
Discord: @jaymesss
```

### Additional Context (Examples, Links, ETC)

Server is running Paper 1.21.1

**Comment by TrentinTheKid** [comment]
We pushed a Lunar Client update to resolve this issue. This issue should be resolved for V2.17.3+ versions of Lunar Client.

Can you confirm this has resolved your issue? @jaymesss

**Comment by jaymesss** [comment]
Issue is resolved, thank you.
